soccer/planning/LineKickPlanner.cpp

```diff
@@ -0,0 +1,201 @@
+#include "LineKickPlanner.hpp"
```

Comment:
Ok, Added some comments. Its "workingish" on simulator. I think I'm really gonna have to work on it with real robots to get more progress so I think its okish for now.